.github/tests/actions/test-setup-helm-docs/action.yml

```
@@ -0,0 +1,14 @@
+name: Test setup-helm-docs
+description: >
+  Just a test for setup-helm-docs
+inputs:
+  helm-docs-version:
+    description: 'Version of helm-docs'
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - uses: ./.github/actions/setup-helm-docs
+      with:
+        version: ${{ inputs.helm-docs-version }}
```

Worth adding a small test to check version?

```suggestion
        version: ${{ inputs.helm-docs-version }}
    - name: Check Helm-docs version
       run: [ $(helm-docs -v | awk '{print $NF}') = ${{ inputs.helm-docs-version }} ]
```